webapp/src/Utils/Utils.php

```diff
@@ -717,6 +718,19 @@ public static function getImageSize(string $filename): array
         return [$width, $height, $width / $height];
     }
 
+    /**
+     * Sanitize the given SVG file contents and return the sanitized SVG file contents.
```

Comment:
I think we don't need the explanation here so I rather have that the name implies this instead of this DOCBLOCK